events/tests/test_notifications.py

```diff
@@ -110,9 +110,9 @@ def test_occurrence_enrolment_notifications_on_model_level(
     user_api_client,
     notification_template_occurrence_unenrolment_fi,
     notification_template_occurrence_enrolment_fi,
-    occurrence,
     project,
 ):
+    occurrence = OccurrenceFactory(id=74, time=now())
```

Comment:
Is there any reason to specify the occurrence id here?

Reply:
Yeah, otherwise the ID might change depending on which tests are run, for example like running `pytest` vs `pytest events`, making the test fail in one of them because it's snapshot is assuming the ID to be exactly some value.

A better fix would probably be to get the IDs out of the snapshots, which could be done at some point when there is too much time.